mavisp/core.py

```diff
@@ -217,11 +220,15 @@ def ingest(self, stop_at='critical'):
             mutations = r['mutations']
             curators = r['curators']
             uniprot_ac= r["uniprot_ac"]
-            refseq_d= r["refseq_d"]
+            refseq_id= r["refseq_id"]
             if mutations is None:
                 mavisp_criticals.append(MAVISpCriticalError("the mutation list was not available, readable or in the expected format"))
             if curators is None:
                 mavisp_criticals.append(MAVISpCriticalError("the metadata file was not available, readable or in the expected format"))
+            if uniprot_ac is None:
+                mavisp_criticals.append(MAVISpCriticalError("the uniprot_ac was not available, readable or in the expected format"))
```

Comment:
this error message is misleading - we never check the format of the uniprot_ac. You can change it to something like "information on Uniprot AC was not available in the metadata file"